samples/src/components/screens/TaskScreen.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component
+    name='TaskScreen'
+    extends='mv_BaseScreen'
+    vm='screens.TaskScreenVM'>
+    <interface>
+	</interface>
+
+    <children>
```

Comment:
Can we get some kind of visual representation that the data is loading?
 - like a button we click to load?
 - a grid that updats when the data is loaded?